docs/source/blog/nereval.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Evaluating Named Entity Recognition parsers with spaCy and Label Studio
+type: blog
+image: ![image](https://user-images.githubusercontent.com/2641205/112475305-f5863900-8d70-11eb-9eb8-9da879f61708.png)
+order: 96
+meta_title: Evaluate NER parsers with spaCy and Label Studio
+meta_description: Use Label Studio to evaluate named entity recognition parsers like spaCy and revise predictions by annotating a gold standard dataset for your data science and machine learning projects. 
+---
+
+![image](https://user-images.githubusercontent.com/2641205/112475305-f5863900-8d70-11eb-9eb8-9da879f61708.png)
+
+This tutorial helps you evaluate accuracy of Named Entity Recognition (NER) taggers using Label Studio. Gather predictions from standard [spaCY](https://spacy.io/) language models for a dataset based on transcripts from the podcast This American Life, then use Label Studio to correct the transcripts and determine which model performed better to focus future retraining efforts.
```

Comment:
should we provide the link to the podcast transcripts here?

Reply:
It's in the steps later, I don't want to distract people up here